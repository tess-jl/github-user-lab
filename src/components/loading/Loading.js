import React from 'react'; 
import loader from './Loading';

const Loading = () => (
  <div className={loader}>Loading...</div>
); 

export default Loading; 
