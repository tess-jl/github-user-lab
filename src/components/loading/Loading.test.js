import React from 'react';
import { shallow } from 'enzyme';
import Loading from './Loading';

describe('Loading component', () => {
  it('renders Loading', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});
