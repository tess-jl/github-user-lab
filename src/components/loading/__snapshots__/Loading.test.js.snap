// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component renders Loading 1`] = `
<div
  className={null}
>
  Loading...
</div>
`;
